apps/gnome-back/src/reports/reports.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { JwtGuard } from "@/auth/jwt/jwt.guard";
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  UseGuards,
+} from "@nestjs/common";
+import { CreateReport } from "./dto/createReport.dto";
+import { ReportsService } from "./reports.service";
+
+@Controller("reports")
+export class ReportsController {
+  constructor(private readonly reports: ReportsService) {}
+
+  @Get("")
+  getAllReports() {
+    return this.reports.getAllReports();
+  }
```

Comment:
tu chyba tylko admin powinien mieć dostęp, ogranicz te endpointy które są dostępne tylko dla admina (rolę chyba da się wyciągnąć z JWTUser. Jeśli nie to będzie to można w osobnym zadanku zrobić